### ISSUE TITLE
Issues with scrobbling tracks (album not sent urlencoded?)

### ISSUE THREAD

**Issue by SHADOW-XIII** [tgwizard/sls#365]
Partly related to https://github.com/tgwizard/sls/issues/272 ? 
We thought it was problem with urlencode of artist but was case of empty/unknown album.
Now it seems it actually might be case of non-urlencode of the Album :)

Tested with 1.5.3, not sure if 1.5.4 has it fixed?

Example tracks that do not scrobble: 
http://ocremix.org/remix/OCR03506 or http://ocremix.org/remix/OCR02997

Most likely the cause is "Album" tag which is: `http://ocremix.org/`
I found it to be the case on my foobar2k desktop client (foo_audioscrobbler plugin)

As per http://www.last.fm/api/scrobbling:
> As with all our write web services, requests must be sent as HTTP POST requests to http://ws.audioscrobbler.com/2.0/ with form urlencoded parameters in the body of the request. The text encoding must be UTF-8.

I believe the URL from Album tag might not be sent as urlencoded parameter (that was case with foobar2k, had to remap album filed there to urlencode and it worked there)

Last.fm support also confirmed that "http://" is also blacklisted, but it works when urlencoded by the looks of it

**Comment by SHADOW-XIII** [comment]
Last.fm support confirmed that "http://" is also blacklisted, but it works when urlencoded by the looks of it

**Comment by SHADOW-XIII** [comment]
catlog using 1.5.3 

we can see the scrobbling is ignored:

```
03-15 09:00:57.717 D/Scrobbler(10730): Scrobbling: Last fm
03-15 09:00:57.721 D/Scrobbler(10730): Retrieved 1 tracks from db: Last fm
03-15 09:00:57.721 D/Scrobbler(10730): Last fm: Track [mTrack=Deus Ex 'Neonature' OC ReMix, mArtist=nervous_testpilot, mAlbum=http://ocremix.org, mDuration=180, mMbId=, mMusicAPI=MusicAPI [clashWithScrobbleDroid=0, enabled=1, id=2, msg=null, name=Poweramp, pkg=com.maxmpz.audioplayer], mQueued=false, mRating=, mRowId=110019, mSource=P, mTimePlayed=0, mTracknr=, mUnknownDuration=false, mWhen=1489568094, mWhenToCountTimeFrom=-1]
03-15 09:00:59.562 D/Scrobbler(10730): Response code: 200
03-15 09:00:59.563 D/Scrobbler(10730): {"scrobbles":{"@attr":{"accepted":0,"ignored":1},"scrobble":{"artist":{"corrected":"0","#text":"nervous_testpilot"},"ignoredMessage":{"code":"1","#text":""},"albumArtist":{"corrected":"0","#text":""},"timestamp":"1489568094","album":{"corrected":"0","#text":"http:\/\/ocremix.org"},"track":{"corrected":"0","#text":"Deus Ex 'Neonature' OC ReMix"}}}}
03-15 09:00:59.566 I/Scrobbler(10730): Scrobble success: Last fm: Ignored Count: 1
```

Oddly enough it says code=1 which should be pointing to "Filtered artist" issue according to audioscrobbler info

Interesting, after some additional testing, it seems that Last.fm is filtering out ALL submissions with http:// - even urlencoded ones :( 

I will check with OCRemix to change their album naming

**Comment by a93-39a** [comment]
I believe this issue has been resolved.
I'll be posting an update soon.
